_shows/20_21/when_sally_met.md

```diff
@@ -0,0 +1,65 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: When Sally Met Gary, Henry, Harry
+playwright: Cecilia Alexander
+student_written: true
+period: Autumn
+season: Online
+season_sort: 410
+date_start: 2020-12-06
+date_end: 2020-12-06
+
+cast:
+- role: Sally
+  name: Rachel Coussins
+- role: Harry
+  name: Demi Idowu
+- role: Chrissy
+  name: Katie Booth
+- role: Flinty
+  name: Charlie Basley
+  
+crew:
+- role: Director
+  name: Cecilia Alexander
+- role: Assistant Director 
+  name: Pete Rouse
+- role: Co-Producer
+  name: Nadia Elalfi
+- role: Co-Producer
+  name: Ylana Gibert
+- role: Shadow Director
+  name: Lauren Birks
+- role: Shadow Producer
+  name: Connor Waldock
+- role: Poster Designer
+  name: Pete Rouse
+- role: Video Editor
+  name: Caetano Capurro
+- role: Graphic Designer
+  name: Zoe Smith
+- role: Trailer Editor
```

Comment:
If we have a trailer can we add it to smugmug and add it here please?

Reply:
I'll ask cecil where it is as I am uncertain :)

Reply:
@samozzy you happy to resolve this if we add an issue to find and add the trailer?

Reply:
Very much so!